ConsoleTests/Program.cs

```diff
@@ -124,7 +124,7 @@ private static void TestIssuesEndPoint(SharpBucketV1 sharpBucket)
             var issue = issuesResource.GetIssue(newIssueResult.local_id);
             var changedIssue = new Issue { title = "Completely new title", content = "Hi!", status = "new", local_id = issue.local_id };
             var changedIssueResult = issuesResource.PutIssue(changedIssue);
-            issuesResource.DeleteIssue(changedIssueResult.local_id);
+
 
             // Issue comments 
             var issueResource = issuesResource.IssueResource(ISSUE_ID);
```

Comment:
We are getting the issue by the `ISSUE_ID`. I don' think that this issue is connected to the `changeIssueResult`.

Reply:
You're creating an `issue` with `ISSUE_ID = 5`, then deleting that same `issue` before trying to post a new `issue comment` on the `issue` you just deleted. That's why I moved the deletion of the `DeleteIssue` to _after_ the `DeleteIssueComment`. ;)

Reply:
`ISSUE_ID` is not used when creating the new issue. This line has the first reference to ISSUE_ID.

The ids are assigned by BitBucket in any case. I guess the `ISSUE_ID` is used to just get some issue that I have manually added to my test repository ;)

Reply:
Oh hahaha silly me. It just so happens to be that this added Issue happens to be ISSUE_ID 5 in my test repository. Thanks for the review :)

Reply:
Yeah, the `Program.cs` file is kind of a mess and can be confusing in some cases. It's just a quick showcase of how to use the API and it could really use some more attention :)